### ISSUE TITLE
Question: how to use jaws.Text?

### ISSUE THREAD

**Issue by belen-albeza** [ippa/jaws#111]
I'm trying to use `jaws.Text`, but I cannot make it work and cannot find any example on how to use it :(

Here's what I've tried so far (here's [the jsfiddle](http://jsfiddle.net/m8qbu/))

``` js
var text;

function setup() {
    text = new jaws.Text({text: "Hello World", x: 10, y:10});
}

function draw() {
    jaws.clear();
    text.draw();
}

jaws.start();
```

What am I doing wrong? Thanks! :)
